.github/issue_label_bot.yaml

```diff
@@ -0,0 +1,6 @@
+# for https://mlbot.net
+label-alias:
+  bug: 'kind/bug'
+  feature_request: 'kind/feature'
+  feature: 'kind/feature'
+  question: 'kind/question'
```

Comment:
does `question` need a template as well? ... maybe to refer to the area/type of question which could be about general process, community related, technical, regarding an area of code, ... just thinking out loud ... or it might be overkill

Reply:
OK I added question as well since I also saw the kubeflow/kubeflow repo has 3 issue templates.